guides/Network nodes/fleek-network-how-to-install-a-node-easily-with-the-assisted-installer.md

```diff
@@ -0,0 +1,204 @@
+draft: false
+hide_title: true
+title: How to install a node easily with the assisted installer
+slug: how-to-install-a-node-easily-with-the-assisted-installer
+image: ./assets/how-to-install-a-node-easily-with-the-assisted-installer.png?202302101751
+date: 2022-11-30T23:00:00.000+00:00
+canonical: ''
+description: Get Fleek Network, is an attempt to make our software more accessible. By providing scripts to automate the installation process of our software, we believe that it can help improve the onboarding experience of our users.
+category: Tutorial
+tags:
+- CDN
+- Guide
+- Getting Started
+- Fleek Network
+---
+
+<!--
+  The following import is intentional (see partial <CheckoutCommitWarning />)
+-->
+import CheckoutCommitWarning from '../partials/_checkout-commit-warning.mdx';
+import Author from '@site/src/components/Author';
+import GitCloneOptions from '../partials/_git-clone-options.mdx';
+
+![Fleek Network: Getting started guide](./assets/how-to-install-a-node-easily-with-the-assisted-installer.png?202302101751)
+
+## Introduction
+
+"Get Fleek Network" is an attempt to make our software more accessible. By providing scripts to automate the installation process of our software, we believe that it can help improve the onboarding experience of our users.
+
+## Pre-requisites
+
+To follow the guide, you will need the following:
+
+- Familiarity with the command-line interface
+- cURL installed
+- A supported Linux Server Operating system (Ubuntu, Debian or ArchLinux)
+- Domain name and permissions to update the DNS Record settings
+
+<CheckoutCommitWarning />
+
+## TL;DR
+
+Open your terminal, connect to your Linux server and run the following command:
+
+```sh
+curl https://get.fleek.network | bash
+```
+
+Follow the steps and have your complete ready quickly! For example, during the process you are required to provide a domain name that should respond with the server where you want the node installed, as the assisted installer will try to secure it.
+
+***
+
+## What is the assisted installer?
+
+The assisted installer is a shell script written in Bash shell scripting language (a language used to interface with the system), that goes through the installation proccess instructed in our guide [Running a node in a docker container](./fleek-network-running-a-node-in-a-docker-container).
+
+Operating system are highly customisable and some users might much prefer to setup the Network Node required applications and dependencies themselves. Some of these users have a lot of system level knowledge, which not all other users have. By putting down the steps programmatically, or in a sequence of commands, we help reduce the frustration that some users might feel otherwise when following guides or longe descriptive processes.
+
+The assisted installer is one of the many options we provide to onboard users of any shape, geographical location, technical knowledge into the Fleek Network, an open network made for everybody.
+
+## How does it Work?
+
```

Comment:
It would be useful (imo) to provide a
1. 
2.
3.
quick breakdown of the steps before diving in, personally for learning it usually makes it easier for me to see the full picture quick and then dive into the steps :) (suggestion)

Reply:
Yeah agree 🙂 Although this one is more of an overview, meaning that the user doesn't necessarily need to learn in advance which steps are going to be executed.